### PR TITLE
fix(install): package status

### PR DESCRIPTION
**Pull request by Mte90**
image

### CODE REVIEW

**Review by b1ek — DISMISSED** [review]

**Comment by Ph0enixKM** [comment]
@Mte90 Can you write code from cloned repo instead of fork? There are sometimes some suggestions that I'd collaborate. For example I think that `250px` looks a bit better (the font size on the package manager table matches the text on the page)

File: docs/getting_started/installation.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ bash -- --user <(curl -s "https://raw.githubusercontent.com/amber-lang/amber/mas
 ```
 - **Package manager**
 
+<div style="width:300px;margin: 0 auto;">
```
Comment (Ph0enixKM) [inline docs/getting_started/installation.md:27]:
```suggestion
<div style="width:250px;margin: 0 auto;">
```

**Review by Ph0enixKM — CHANGES_REQUESTED** [review]

**Comment by Mte90** [comment]
Very strange you should be allowed to edit the fork as I enable all the PR:

image


Anyway as you prefer I always keep a fork so the branch doesn't became a mess.

**Comment by Ph0enixKM** [comment]
Oh okay then

**Review by Ph0enixKM — APPROVED** [review]
LGTM

File: docs/getting_started/installation.md [hunk 2]
```diff
@@ -46,6 +48,16 @@ The package is avaiable as `amber-lang` on [nixpkgs](https://github.com/NixOS/ni
...
 
+## Snap
+
+You can install the snap package in any distro via the store:
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/amber-bash)
+
+or from the terminal:
+```bash
+sudo snap install amber-bash --classic
+```
```
Comment (b1ek) [inline docs/getting_started/installation.md:59]: it doesnt have to be that overworded

``````suggestion
## Snap
[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/amber-bash)

```bash
sudo snap install amber-bash --classic
```
``````

Comment (Ph0enixKM) [reply docs/getting_started/installation.md:59]: @Mte90 Can you take a look?

**Review by Ph0enixKM — APPROVED** [review]
LGTM

**Pull request merged**